db/seeds.rb

```diff
@@ -5,5 +5,49 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
-role = Role.create!([{name: 'organizer'}])
-user = User.create!([{name: 'Wolverine', roles: role, github_handle: 'Adminis28'}])
\ No newline at end of file
+
+
+# Temp commented out seeds from upstream repo
+  # role = Role.create!([{name: 'organizer'}])
+  # user = User.create!([{name: 'Wolverine', roles: role, github_handle: 'Adminis28'}])
+
+require 'FFaker' #TODO: do I need this?
```

Comment:
Bundler will take care of requiring the gem for you, so you shouldn't need to explicitly do it.